api4/file.go

```diff
@@ -97,9 +95,6 @@ func multipartReader(req *http.Request, stream io.Reader) (*multipart.Reader, er
 }
 
 func uploadFileStream(c *Context, w http.ResponseWriter, r *http.Request) {
-	// Drain any remaining bytes in the request body, up to a limit
-	defer io.CopyN(ioutil.Discard, r.Body, maxUploadDrainBytes)
```

Comment:
I am still not sure why this was introduced but it can cause problems when returning errors since we will have to wait until all that data has came in.

Reply:
This was a while ago, during my sim project so I don't have a clear recollection. There might have been an issue in the tests without it? Anyway, it doesn't appear necessary, and it's in the client anyway; remove it.